ludwig/data/postprocessing.py

```diff
@@ -54,7 +54,7 @@ def postprocess_batch(df):
             )
         return df
 
-    predictions = backend.df_engine.map_batches(predictions, postprocess_batch)
+    predictions = backend.df_engine.map_batches(predictions, postprocess_batch, enable_tensor_extension_casting=False)
```

Comment:
Is the intention for this to always be `False`, or are you planning on setting this conditioned on the output feature type?

Reply:
Hm, I was thinking of it always being `False` so that we would have predictable/unified behavior across all output features after postprocessing. What do you think?